roles/openshift_preflight/common/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# check content available on all hosts
+- when:
+    - not containerized
+  block:
+
+    - name: determine if yum update will work
+      action: check_yum_update
+      register: r
+
+    - set_fact:
+        oo_preflight_check_results: "{{ oo_preflight_check_results + [r|combine({'_task': 'check content available on all hosts'})] }}"
```

Comment:
I'm not sure we can reliably use this method of updating variables in this manner.

Instead of redefining and updating oo_preflight_check_results in all playbooks, what if there was a single variable per role to track the results and the playbook aggregated the result variables and determined pass/fail?